src/modules/modal/components/modal-view/modal-view.jsx

```diff
@@ -75,10 +76,13 @@ export default class ModalView extends Component {
           {p.modal.type === MODAL_NETWORK_DISCONNECTED &&
             <ModalNetworkDisconnected {...p} />
           }
+          {p.modal.type === MODAL_ACCOUNT_APPROVAL &&
+            <ModalApproval {...p} />
+          }
           {p.modal.type === MODAL_ESCAPE_HATCH &&
             <ModalEscapeHatch {...p} />
           }
-          {p.modal.canClose &&
+          {p.modal.canClose && p.modal.type !== MODAL_ACCOUNT_APPROVAL &&
```

Comment:
Is the intention for the account approval modal to be un-closeable?  I think it would be better to allow it to be closed, and just not submit the order ticket if the user closes the modal.

Reply:
It's actually a styling issue im side-stepping. The account approval modal is always closeable, it has a close button built into it and this was done to allow for side by side buttons for approve/close, instead of approve then a line break, close. I could remove this and just have the linebreak, but side-by-side seemed to be the better solution, so in this scenario im just ensuring that we don't have 2 close buttons. I'll discuss this quickly with @sharkcrayon to see if I shouldn't bother with side-by-side buttons and just leave it as stacked.

Reply:
Ah, got it.  As long as it's closeable, fine by me.